tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
```

Comment:
Let's leave the __init__.py files empty for now.

Also, could you move `tests` underneath the `pybluedot` directory you created? This way we can refer to it as part of the project path (i.e. `pybluedot.tests`